FloripaPublicTransportation/MapViewController.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  MapViewController.swift
+//  FloripaPublicTransportation
+//
+//  Created by Marcelo Gobetti on 2/22/16.
+//  Copyright © 2016 Marcelo Gobetti. All rights reserved.
+//
+
+import MapKit
+
+class MapViewController: UIViewController, MKMapViewDelegate, CLLocationManagerDelegate {
+    @IBOutlet weak var descriptionLabel: UILabel!
+    @IBOutlet private weak var mapView: MKMapView!
+    @IBAction private func addPin(sender: UILongPressGestureRecognizer) {
+        guard sender.state == .Began else {
+            return
+        }
+        
+        self.mapView.removeAnnotations(self.mapView.annotations)
+        let touchCoordinates = self.mapView.convertPoint(sender.locationInView(self.mapView), toCoordinateFromView: self.mapView)
+        let annotation = MKPointAnnotation()
+        annotation.coordinate = touchCoordinates
+        
+        let geocoder = CLGeocoder()
+        geocoder.reverseGeocodeLocation(CLLocation(latitude: annotation.coordinate.latitude, longitude: annotation.coordinate.longitude), completionHandler: {
```

Comment:
this takes a little bit of time to process, at first I clicked twice in the screen trying to see what would happen, there were no indication to a loading state can you explain the reasoning behind not adding a loading state?